src/cdk/a11y/focus-trap/focus-trap.ts

```diff
@@ -26,6 +26,15 @@ import {take} from 'rxjs/operators';
 import {InteractivityChecker} from '../interactivity-checker/interactivity-checker';
 
 
+/** Options for how to handle focus upon opening the dialog */
+export const enum AutoFocusOption {
```

Comment:
In an ideal world, this enum would live in `cdk/dialog`. Unfortunately, since cdk/dialog doesn't exist yet, I think we should duplicate this type for each component implementing `autoFocus` since it's not strictly related to focus trap